mitodl/mitxpro
# #442 Product Page: Propel your career section

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #442 

#### What's this PR do?
Adds a text section (example: "Propel your career" on product page) with an optional call to action button. The action button title and link are configurable, and does not appear if either the link or the title is not configured. The section also supports light/dark theme. 

#### How should this be manually tested?
Add a `TextSection` type subpage to a program or course page. The text section should appear under the FAQs section as per the design specs.

#### Screenshots (if appropriate)
![Screenshot from 2019-06-10 16-42-24](https://user-images.githubusercontent.com/45350418/59194353-32843a00-8ba2-11e9-96bd-0694efd56ae5.png)
![Screenshot from 2019-06-10 17-02-58](https://user-images.githubusercontent.com/45350418/59194358-37e18480-8ba2-11e9-81ee-31c8465aa8fe.png)
![Screenshot from 2019-06-10 17-02-13](https://user-images.githubusercontent.com/45350418/59194367-3d3ecf00-8ba2-11e9-8795-7ad8785edf8f.png)
